src/main/java/UI.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class UI {
+    private Scanner scanner;
+
+    public UI() {
+        scanner = new Scanner(System.in);
+    }
+
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    public void showMessage(String message) {
+        System.out.println(message);
+    }
+
+    public void displayError(Exception e) {
+        System.out.println(e.getMessage());
+    }
+
+    public void showLogo() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        showMessage("Hello from\n" + logo);
+    }
+
+    public void showEndMessage() {
+        showMessage("Bye");
```

Comment:
Since no commands are expected after this, it could be a good point to close the scanner.
(I know it's unnecessary since the program is being closed, but my IDE does give a very annoying error regarding this)